manticore/core/manticore.py

```diff
@@ -278,6 +283,69 @@ def _join_workers(self):
             while len(self._workers) > 0:
                 self._workers.pop().join()
 
+    #
+    # Symbol resolution
+    #
+
+    def _addr_by_symline(dwarfinfo, symbols, line):
```

Comment:
How does one fetch this information (dwarf line for a given symbol)?

I am afraid it might not be that used by the users :S.

Reply:
`pyelftools` exposes an API to DWARF that can accurately identify the function's actual line number and the file that it comes from, which I believe this is made available through the Line Number Table. [This example](https://github.com/eliben/pyelftools/blob/master/examples/dwarf_decode_address.py) best shows this.